cookbooks/bcpc-hadoop/recipes/hive_config.rb

```diff
@@ -129,6 +129,8 @@
     'hive.server2.authentication.ldap.Domain' =>
       hive_site_vars[:hs2_ldap_domain]
   }
+else
+  hs2_auth_values = {}
```

Comment:
if no security is specified a blank hs2_auth_values is declared so that https://github.com/cbaenziger/chef-bach/blob/0b59a8ca80818836d450e3ff745e977127f419c9/cookbooks/bcpc-hadoop/recipes/hive_config.rb#L136 does not break